project-base/translations/messages.cs.po

```diff
@@ -31,6 +31,9 @@ msgstr "Uplatnit"
 msgid "Attention! You are administrator logged in as the customer."
 msgstr "Pozor! Jste jako administrátor přihlášen za zákazníka."
 
+msgid "Availability"
+msgstr ""
```

Comment:
could you please translate these messages?

Reply:
fixed